webknossos/webknossos/geometry/nd_bounding_box.py

```diff
@@ -209,11 +209,11 @@ def with_color(self: _T, color: Optional[Tuple[float, float, float, float]]) ->
         Returns a new instance of NDBoundingBox with the specified color.
 
         Args:
-        - color (Optional[Tuple[float, float, float, float]]): The color to set for the bounding box.
+            color (Optional[Tuple[float, float, float, float]]): The color to set for the bounding box.
             The color should be specified as a tuple of four floats representing RGBA values.
```

Comment:
```suggestion
                The color should be specified as a tuple of four floats representing RGBA values.
```
The plugin seems to need an indentation here to recognize value and description.

Reply:
Looks like Google-style doc string require another indent for multi line descriptions 

https://stackoverflow.com/questions/58930609/how-to-format-long-lines-of-argument-description-in-google-docstring-style